### ISSUE TITLE
"Überall abladen" (zB für Rübenmieten am Feldrand) integrieren

### ISSUE THREAD

**Issue by XPiRiJenZ** [Stephan-S/FS22_AutoDrive#159]
Grüße zusammen!
Da man bei CP raus hört, dass Transporte mit AD abgewickelt werden sollen drängt sich mir die Frage auf, ob man nicht ein "Überall abladen" mit integrieren kann (wie STRG + i im Standardspiel), um zB Rübenmieten am Feldrand anzulegen.
Die Möglichkeit hierzu bestand im LS19 über CP und würde jetzt im 22er gänzlich wegfallen.

Danke für eure Mühe, dass ihr es uns so schnell ermöglicht habt, AD im 22er zu nutzen! :)

**Comment by Iwan1803** [comment]
Auf "hören sagen" gebe ich erstmal nix. Ich stelle mir aber dennoch die Frage, welchen Vorteil das "lade hier ab" bringen soll.
Bei Rübenmieten nur, um die Maus mal nutzen zu können?
Das Problem ist ja: was ist, wenn an der Stelle schon ein Berg aus Rüben liegt? Soll AD dann irgendwie versuchen, dort noch abzuladen oder wie stellst du dir das vor?

**Comment by XPiRiJenZ** [comment]
Vorab: ich selbst nutze keine Maus und wollte lediglich anregen, diese Funktion mit zu integrieren, da sicher einige aus der Comm das vermissen könnten.
Der Vorteil einer Miete, ganz gleich ob Rüben, Häckselgut oder Hackschnitzel ist doch, dass man weniger Fahrzeuge zum Abfahren braucht und die Erntemaschinen ohne große Downtime am Laufen halten kann.
Der Mechanismus müsste dann ähnlich wie im 19er-CP sein, dass es einen Startpunkt für die Miete gibt und sich die Miete zB entlang eines AD-Kurses erstreckt (bei Abkippseite "links/rechts" ist es sicherlich einfacher umzusetzen, als bei Abkippseite "hinten")
...und am Ende, wenn du es so willst, ja, dann auch "nur, um die Maus mal nutzen zu können" ;)

**Comment by Iwan1803** [comment]
Zum einen seh ich immer noch nicht wirklich einen Sinn darin und zum anderen auch nicht, warum wir etwas "nachbauen" sollen, was mal mit CP ging und vielleicht auch wieder gehen wird.
Mal abwarten, ob jemand anders noch andere Vorteile aufzeigen kann, die ich vielleicht einfach gerade nicht sehe 😉

**Comment by andreasthu** [comment]
Hallo Iwan
ich werde sicher nie Rübenmieten anlegen, noch die ropa maus benutzen.
sehe da auch keinen sinn dahinter, Rüben und Kartoffel werden entweder eingelagert oder gleich verkauft. 
Erstmaligen Sinn sehe ich bei sowas um z.b. was an Boden abkippen zu können, was dann von dem Lizard S-710 Förderband aufgenommen wird um das anschließend in eine Schüttguthalle zu verfrachten.
der 2. (meiner Meinung nach Sinnvoller als pkt.1) wäre als eine "hier abkippen" Funktion für die Ballensammelwagen, um endlich auch Ballen am Trigger verkaufen zu können.
lg
andreas

**Comment by blubstar86** [comment]
Guten Tag,
ich würde "überall abladen" jedenfalls richtig feiern, dann kann man mit Rüben oder etwas anderem endlich mal so richtig gas geben im SP.
Finde es irgendwie befriedigend es erst irgendwo hinzukippen um es dann später wieder aufladen zu müssen, Geschmäcker sind eben verschieden.

Grüße

**Comment by Terrabey** [comment]
Nieten wäre eine tolle Sache vor allem dann wenn man große Felder zu Ernten hat

**Comment by rohne83** [comment]
Grade für Leute die wie ich trotz höchstmöglicher Automatisierung auf realistischen Betrieb achten, ist eine Kartoffel-/Rübenmiete natürlich notwendig. Wobei ich aber die Hoffnung noch nicht aufgegeben habe, dass CP das zumindest in den Überladewagen mit einbaut. Von daher würde ich dem vorerst bei AD noch nicht zwangsweise eine hohe Priorität zumessen.

**Comment by Holzotte** [comment]
Hallo, 
ich denke es würde schon reichen wenn in diesem Fall, AutoDrive die Ropa Maus als Drescher erkennt und ihr beim überladen folgt wie einem richtigen Drescher. Dann kann man sich das überall abkippen sparen. Damit wäre denjenigen die auf realistische Spielweise setzen schon geholfen. Die Frage ist nur wie kompliziert es wäre das umzusetzen.

**Comment by rohne83** [comment]
nope, damit hat das gar nix zu tun. Von der Maus sich beladen lassen kann AD schon jetzt, wenn auch recht umständlich. Das große Problem zur Zeit ist überhaupt erstmal das Anlegen der Kartoffel/Rübenmiete. Dafür braucht es zwingend überall abladen, weil ja auf den Feldboden gekippt werden muss.
Ist aber erstmal bei CP, da gibts auch schon ein Issue zu (https://github.com/Courseplay/Courseplay_FS22/issues/606), zumindest für den Roder selbst un den Überladewagen.

**Comment by rohne83** [comment]
Aufräumpotenzial: #670 und #611 drehen sich grundsätzlich um genau das selbe Thema, nur mit anderen Fallbeispielen.
